recipes/metview-python/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "metview-python" %}
+{% set pypi_name = "metview" %}
+{% set version = "1.7.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ pypi_name[0] }}/{{ pypi_name }}/{{ pypi_name }}-{{ version }}.tar.gz
+  sha256: 67e4ebe203f8249f42e87a06fda432bfc6767fc50993575f4f4fb9fddffe464c
+
+build:
+  number: 0
+  noarch: python
+  script: '{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv '
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+    - pytest-runner
+  run:
+    - cffi
+    - cfgrib
+    - numpy
+    - pandas
+    - python >=3.6
+    - xarray
+
+test:
+  commands:
+    - test -f $(python -c "import site; print(site.getsitepackages()[0])")/metview/__init__.py  # [not win]
+
+about:
+  home: https://github.com/ecmwf/metview-python
+  license: Apache-2.0
+  license_family: Apache
+  license_file:
+    - PKG-INFO
```

Comment:
This should be `LICENSE` (based on the PyPI tarball)

Reply:
Quite right, I've fixed it now. Thanks Ryan (@dopplershift) !